### DISCUSSION TITLE
Model engine entity looks black

### DISCUSSION THREAD

**Discussion by HotsukokuAito** [Kas-tle/java2bedrock.sh#29431] [Converter Help]
I have converted a model engine resource pack, and it seems that even animations work. But there's a problem: The texture can be seen only if you hit the entity and for a few seconds.
![Screenshot_2024-07-02-15-04-06-186_com mojang minecraftpe](https://github.com/Kas-tle/java2bedrock.sh/assets/148512520/7ac9c92c-dcae-4f80-a695-c1f66a1b87ad)

**Comment by diandlt** [comment]
Try using geyseroptionalpack :)
